lib/content-services/src/lib/content-node-share/content-node-share.dialog.html

```diff
@@ -31,7 +31,7 @@
                     [style.display]="isExpiryDateToggleChecked ? 'block' : 'none'"
                     data-automation-id="adf-slide-toggle-checked"
                     class="adf-share-link__date-time-container">
-                    <mat-form-field class="adf-full-width adf-float-label" floatLabel='always'>
+                    <mat-form-field class="adf-full-width adf-float-label" floatLabel='never'>
```

Comment:
For anyone who will be looking at that PR in the future. I changed it because of strange workaround applied in that file and removed: 

https://github.com/Alfresco/alfresco-content-app/pull/3368/files#diff-13b18163e00bbf172ccac6d5bb175406262493bd5d3f7cabb7be2b5f557d494b

Somebody was hiding labels using css styles instead of mat form API. So then they were not visible in that modal: 
<img width="770" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/85e3e2bd-ab4d-4077-af12-6ce187349ff7">